backend/uclapi/workspaces/occupeye/cache.py

```diff
@@ -625,6 +625,17 @@ def feed_cache(self, full):
         )
         self._redis.set(last_modified_key, current_timestamp)
 
+        if not settings.DEBUG:
+            url = settings.CACHET_URL
+            url = url+"components/3"

```

Comment:
```suggestion
            url = "{}/components/3".format(settings.CACHET_URL)
```
Might be prettier

Although again it might be good to extract out component 3 into a settings variable somewhere because that can change if we update Cachet.